statistics/handle/handle.go

```diff
@@ -330,13 +330,15 @@ func (h *Handle) MergePartitionStats2GlobalStats(sc *stmtctx.StatementContext, i
 	allHg := make([][]*statistics.Histogram, globalStats.Num)
 	allCms := make([][]*statistics.CMSketch, globalStats.Num)
 	allTopN := make([][]*statistics.TopN, globalStats.Num)
+	allID := make([]int64, 0, globalStats.Num)
+	allFieldType := make([]*types.FieldType, 0, globalStats.Num)
```

Comment:
Since these two variables are not used in this function, why create them?

Reply:
I forget to delete them. It's useless.